### DISCUSSION TITLE
how can i make one like this help me please witch programming language i need to use , Or if i make in html and capture it how it is ?

### DISCUSSION THREAD

**Discussion by RV2003331** [obsproject/obs-studio#10063] [General]
![(29) Cricket Scoring Software with Replay Function - YouTube - Google Chrome 02_01_2024 18_48_37](https://github.com/obsproject/obs-studio/assets/149759881/74b53780-2ffd-459c-ab5d-d2762454148a)

**Comment by RV2003331** [comment]
Txt output Creating I think

**Comment by Lain-B** [comment]
Duplicate of #10064
